pkg/server/catalog/catalog.go

```diff
@@ -275,11 +271,8 @@ func loadSQLDataStore(ctx context.Context, log logrus.FieldLogger, datastoreConf
 		return nil, fmt.Errorf("pluggability for the DataStore is deprecated; only the built-in %q plugin is supported", ds_sql.PluginName)
 	}
 
-	ds := ds_sql.New()
-	ds.SetLogger(common_log.NewHCLogAdapter(log, telemetry.PluginBuiltIn).Named(sqlConfig.Name))
-	if _, err := ds.Configure(ctx, &spi.ConfigureRequest{
-		Configuration: sqlConfig.Data,
-	}); err != nil {
+	ds := ds_sql.New(log.WithField("subsystem_name", sqlConfig.Name))
```

Comment:
can you use telemetry.SubsystemName here?

Reply:
:+1: